packages/elements/src/react/common/form/index.tsx

```diff
@@ -149,7 +166,7 @@ const useForm = ({ flowActor }: { flowActor?: BaseActorRef<{ type: 'SUBMIT' }> }
 
   return {
     props: {
-      [`data-${validity}`]: true,
+      ...(errors.length > 0 ? { 'data-global-error': true } : {}),
```

Comment:
It didn't make much sense to me to have `data-valid`/`data-invalid` on the `<form>` if that's already placed further below in the DOM tree. And in those instances we refer to the `<FieldError>` state, not `<GlobalError>`.

So renaming this to make it more clear that this is about global errors.